### ISSUE TITLE
Finish: Content Management contribution (DoD)

### ISSUE THREAD

**Issue by iJungleboy** [oqtane/oqtane.docs#83]
A lot was contributed recently by @thabaum and it's been included. 

But: there are a bunch of issues which must be resolved.

These issues would be apparent if the solutions was simply compiled once using VS-Code.

1. Build shows a lot of warnings because broken links
2. Many of the pages don't appear anywhere, because they are not mentioned in any TOC

I could of course try to fix all this myself, but it's really important that contributions work and don't cause additional effort upstream.
So it's important that all contributors actively use the tools.

Thanks!

**Comment by thabaum** [comment]
@iJungleboy I have a lot more I have done, I have set this as "Draft" as I am going to propose a much bigger change/update to our docs, completely reorganizing the Guides and Manuals sections.

It is a huge job but I am making progress daily when I can and have it set as a priority.  I am using Visual Studio Code with utilizing the Preview section... but need to get familiar with the PUSH/PULL to github as this PR is bugged out that I have in the repo.  

The links would otherwise... work great from my original update, but since did a lot of changes and set it as "DRAFT" DO NOT MERGE status until I can get these latest updates, and updates to come later today.

For example, I am moving EVERYTHING that is NOT core related and more like a "GUIDE" to the guides section.  THis includes installation docs, modules, themes and so on that are for developing and such.

I just want to make the manuals section focus strictly on the CORE part of using Oqtane.  For installation for example... just the Installation Wizard and basic setup instructions with links going to the guides.

As you mentioned this was looking more complex than it really is and may overwhelm a person reviewing our docs for the first time.

It will make sense... I will push it soon maybe tonight if I can get a conflict resolved in that branch.  I broke a lot of things while working in GItHub which I should of just closed the PR so you had not gotten confused as you are.

It is just a review for things to come I was sharing... but should of just shared a link to my repo in the issue section.

Again, this is a complete ADMIN/HOST/CONTENT/INSTALLATION/GUIDES rewrite of the entire docs.  it was just too confusing to navigate what goes where... this will clear up a lot of confusion I hope you will follow and understand once I get to that point of sharing A LOT of progress made.

It will be night/day - then you can add some final touches on it such as navigation and other things at the bottom and sides of pages to help flow through the docs contributed is what I am hoping we can work out together.

Then Greg or whoever will be able to enhance these all easier as I am hoping to also put to bed all issues in the issue section at the same time with a start.  All but the issues I have not created so we don't have any showing but those 2.

Sorry for confusion, you are correct you should work with Visual Studio Code or whatever... but you missed my messages relating and I understand I should of just closed the PR and re-created it once it was all prepared as I made a major PIVOT once I started working in the other areas I wasn't originally going to work on (Greg was going to)

I have a large number of new images...
As for a new TOC:

Manual Section:
Installation - Core Installation Manual
Content Management - Core Content Management Manual
Admin Management - Core Admin Management Manual
Host Management - Host Management Section
Troubleshooting - Refers to the Guides section and all troubleshooting docs related.

Guides:
Concepts - Guides for concepts working with Oqtane
Extensions Development - ...
Installation - ...
Modules - Guides for modules
Themes - Guides for themes
Marketplace - Guides for marketplace
Troubleshooting - Guides for Troubleshooting
Roadmap & History - Roadmap information

I see a lot has been pushed already... hopefully this all makes sense as I may have to copy all to a new PR form the looks of it
Documentation - Guides for working with documentation

So I will make another push here shortly once I figure out how to proceed.

Thanks!

**Comment by thabaum** [comment]
@iJungleboy looks like it is full steam ahead, and the idea passed - I will make a new PR today, and more each day so you can continue to merge/fix what you can as you see fit.

**Comment by thabaum** [comment]
@iJungleboy I will make one last push in a few hours once I complete the Host Management section to a point.  I have the admin management and content management sections done and have fixed a lot of the guildes section so we can then review it in the docs closer over the next few weeks for the next meeting.  I will post the PR when I am done, please hold off on merging until then.

**Comment by thabaum** [comment]
I am hoping the "Content Management" section has been resolved, we need to finish up the Host Management and Guides to reflect all the updates to the new TOC layout.  I did a lot of that in the last PR, but there are still room for improvements.

I did complete the Content Management and Admin Management to a level.  I will fine tune these areas as I visit them and make links between docs when different topics are discussed in a certain doc that can also help make the doc more useful.

Just doing the basic layouts of the core framework should be huge, and moving all the how to stuff to "Guides" should also help and we can build more links between them as well to help people move through a topic of interest.

As for the Host Management section I will work on this in detail next.  I did get all the layout of them done, but there is a lot of missing information that still needs updated.

Some features in Admin Management still need pages in Host as they have a few added features in Host role.  

Overall this really should clear up a lot of confusion inside the manual and set clear paths forward for the Guides and Manuals documentation.

Thank you for building and fixing things as needed @iJungleboy.  Talk more at next meeting!

**Comment by thabaum** [comment]
fix git and github... like damn WTF who wants to do any contributions I can see why it is like only super geeks can use it.  Anything logical is out the window.  It is designed for the super geeks who actually get paid to use it!

This whole process has me pissed off lol I really want to give up, but I wont... so many issues to overcome even if you show me for an hour, I will run into more issues.

I just see so many changes lost, so much work gone, it is hard to bare.

So many different ways to skin this thing... but if you mix two processes it spoils the entire process.

I liked codespaces working with files and commits, they actually made all my file changes, this other way is SUPER geek way to add or change file names.  I am doing 100 things wrong I am sure but I dont get paid to do this!

Seriously I feel like tossing it in the corner and forgetting about it...  feel enough like a dumbass to want to continue lol what a joke this entire process/system is for something so straight forward and simple to want to do just to... help.

I will get this down, I have a user manual I am creating to help me get unstuck I will submit for #87 hopefully.  I should be done 100 times over again, lost so much work I have no clue what was done/not done now.

I dont want to lose what I did on my recent branch so I will push that as a PR now... I did this the way I now I wont F it up.

Then maybe I can create a new branch with the other fixes I am working on is my guess so please merge this next PR or two I will create SLOPPY and crappy workflows of testing out what will actually get me working like I want to in order to assist here.

**Comment by thabaum** [comment]
@iJungleboy #88 and #89 cure all the issues.  I used a combination of the IDE and this website to get it done.

I will keep working on my workflow, I just got a few things to pan out.  I hate going backwards too many times redoing things I had done already.

If you can please merge those two they should resolve the 50 warnings and add a lot of great new stuff to the docs as far as troubleshooting guides and stuff like migrating to start those sections as well.

I will focus on .NET 9 #85 once it is released to help make it build as needed.  I think I resolved issues for .NET 8 in the last upgrade so maybe refer to that era of PR's for clues.

**Comment by thabaum** [comment]
@iJungleboy thanks for merging those, I see 27 errors that need resolved I will fix now that i can build this.

**Comment by thabaum** [comment]
@iJungleboy I resolved the remaining issues, and am adding more great stuff... I may even tackle all issues today except the latest ones I would like to discuss further at our next meeting.  This way we can put to bed the issues logged and get down to just some of the new ones.

Of coarse all of the docs I will be asking anyone to help improve further, even rewrite them to enhance them so they are the best they can be... you wont hurt my feelings if something gets moved/updated or even removed.

**Comment by thabaum** [comment]
@iJungleboy can close with #91 ?!
